catt/subs_info.py

```diff
@@ -4,32 +4,60 @@
 
 from .error import SubtitlesError
 from .util import create_temp_file
+from .util import guess_mime
 
 
 class SubsInfo:
     """
     This class facilitates fetching/reading a remote/local subtitles file,
     converting it to webvtt if needed, and then exposing a path to a tempfile
     holding the subtitles, ready to be served.
-    An url to the (expected to be) served file is also exposed. The supplied
+    An url to the (expected to be) served file is also exposed.
+
+    For local subtitle files, the url points to this computer. The variable
+    self.local_subs will be True; CATT will detect this and start a web
+    server so that ChromeCast can fetch it. Only devices in the same network
+    can access this file.
     local_ip and port params are used for this purpose.
```

Comment:
Is this forgotten from the previous docstring?

Reply:
I'm sorry I'm late. The answer's yes and no.

`self.local_subs` is still used, but now it's not directly provided by `self._read_subs()`. instead, `self._read_subs()` provides a MIME type only if the subtitles file was originally remote, AND the script uses the existence of the MIME type to infer if the file was remote or not.

_This_ behaviour isn't documented anywhere, though, so I'll update the docstring right away.